app/assets/javascripts/controllers/container_dashboard/container_dashboard_controller.js

```diff
@@ -61,7 +61,7 @@ miqHttpInject(angular.module('containerDashboard', ['ui.bootstrap', 'patternfly'
           id = '';
         }
         else {
-          id = '/'+ (/ems_container\/show\/(\d+)/.exec($location.absUrl())[1]);
+          id = '/'+ (/\/(\d+)/.exec($location.absUrl())[1]);
```

Comment:
@zeari we changed the routes. 
the route was: `/ems_container/show/<:id>`
the route now is: `/ems_container/<:id>`
since correct roude can have only once a `"/<number>" `sequence the new regex will work for the new routes.